### ISSUE TITLE
.table-bordered rules on react-bootstrap-table.css should be scoped to .react-bs-table

### ISSUE THREAD

**Issue by maniqui** [AllenFang/react-bootstrap-table#560]
This set of rules on `react-bootstrap-table.css` (and its minified versions, of course) affect all tables on a BS project, even those that are not created/managed by react-bootstrap-table. That's not good.

So, all the selectors in the next code block should be prefixed with `.react-bs-table`.

 ```css
.table-bordered {
    border: 0px;
}
.table-bordered > thead > tr > th,
.table-bordered > thead > tr > td {
  border-bottom-width: 2px;
}
.table-bordered > tfoot > tr > th,
.table-bordered > tfoot > tr > td {
  border-top-width: 2px;
}
.table-bordered > tfoot > tr > th,
.table-bordered > tfoot > tr > td {
  border-bottom-width: 0px;
}
/*Esquerda*/
.table-bordered > thead > tr > th:first-child,
.table-bordered > tbody > tr > th:first-child,
.table-bordered > tfoot > tr > th:first-child,
.table-bordered > thead > tr > td:first-child,
.table-bordered > tbody > tr > td:first-child,
.table-bordered > tfoot > tr > td:first-child {
  border-left-width: 0px;
}
/*Direita*/
.table-bordered > thead > tr > th:last-child,
.table-bordered > tbody > tr > th:last-child,
.table-bordered > tfoot > tr > th:last-child,
.table-bordered > thead > tr > td:last-child,
.table-bordered > tbody > tr > td:last-child,
.table-bordered > tfoot > tr > td:last-child {
  border-right-width: 0px;
}
/*Topo*/
.table-bordered > thead > tr:first-child > th,
.table-bordered > thead > tr:first-child > td {
  border-top-width: 0px;
}
/*Baixo*/
.table-bordered > tfoot > tr:last-child > th,
.table-bordered > tfoot > tr:last-child > td {
  border-bottom-width: 0px;
}
```

**Comment by maniqui** [comment]
Generally speaking, I think the CSS rules for react-bootstrap-table shouldn't provide any extra aesthetics,  like rounded corners and that sort of stuff. The reason is that it makes it harder to make it look like vanilla/default Bootstrap tables if react-bootstrap-table applies its own set of aesthetic changes.

**Comment by AllenFang** [comment]
@maniqui, I'll fix it in these day. Thanks :)

**Comment by AllenFang** [comment]
Fixed on ```v2.4.1```. Sorry for lately fixing :)
